tools/pirlo-tests.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+set -e
+echo "Downloading pirlo-cli tool"
+curl https://pirlo-downloads.s3-us-west-1.amazonaws.com/pirlo-cli/pirlo-cli -o pirlo-cli
```

Comment:
This is fine for now for getting started, but one thing that would streamline things is to have the CLI tool available as a package on NPM, so that we could add it to our dependencies in `package.json` and it'd get installed by `yarn install`.

In particular that'd mean that once it's downloaded, it doesn't have to be re-downloaded unless a new version is specified. Doesn't much matter in CI, but that would make it a lot smoother to use a script like this interactively.

(Despite the name, there's no need for JS let alone Node to be involved in an NPM package; it basically amounts to a tarball with a name and a version number and a few other bits of metadata, all stuffed into a JSON file at a well-known name inside the tarball. Also it's shockingly easy to upload to NPM.)

Reply:
Interesting idea. Will definitely add this to our development roadmap and implement it as soon as possible.